atom/common/trace_events.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2018 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#ifndef ATOM_COMMON_TRACE_EVENTS_H_
+#define ATOM_COMMON_TRACE_EVENTS_H_
+
+#define TRACE_ELECTRON0(name) TRACE_EVENT0("electron", name)
+#define TRACE_ELECTRON1(name, arg1) TRACE_EVENT0("electron", name, arg1)
+#define TRACE_ELECTRON2(name, arg1, arg2) \
```

Comment:
I don't think we need to define custom macros here, it's fine to just write `TRACE_EVENT0("electron", ...)` etc. directly.